compiler/src/iree/compiler/Codegen/Common/Passes.td

```diff
@@ -529,6 +529,11 @@ def PadDynamicAllocPass :
 def PolynomialApproximationPass :
     Pass<"iree-codegen-polynomial-approximation", ""> {
   let summary = "Convert math operations to their polynomial approximation";
+  let options = [
+    Option<"isLLVMGPU", "is-llvmgpu", "bool",
```

Comment:
`isLLVMGPU` seems OK, but this is not a great solution. Id rather make this a list of strings for example that you can use to just pass the operation name and used that instead. But this is fine for now

Reply:
actually, I'm going to upgrade that to a request for change here - we're going to find different LLVMGPU targets (cuda/rocm/etc) and different architectures for each are likely to have different sets of optimal functions to approximate. not hard to make it a list now and not have to deal with the mess later.

Reply:
Got it. It makes sense to exclude architecture dependent code from the common ones.

 I have updated the code to use a list of op names to ignore.